pandas/tests/reshape/test_cut.py

```diff
@@ -87,7 +87,9 @@ def test_bins_from_interval_index_doc_example():
     # Make sure we preserve the bins.
     ages = np.array([10, 15, 13, 12, 23, 25, 28, 59, 60])
     c = cut(ages, bins=[0, 18, 35, 70])
-    expected = IntervalIndex.from_tuples([(0, 18), (18, 35), (35, 70)])
+    expected = IntervalIndex.from_arrays(
+        left=np.array([0, 18, 35]), right=np.array([18, 35, 70])
```

Comment:
not a big deal but why is this necessary?

Reply:
Yeah, not needed after #50195. I'll revert.